lib/api/bucketGet.js

```diff
@@ -154,13 +173,28 @@ function processMasterVersions(bucketName, listParams, list) {
     const isTruncated = list.IsTruncated ? 'true' : 'false';
     const xmlParams = [
         { tag: 'Prefix', value: listParams.prefix || '' },
-        { tag: 'Marker', value: listParams.marker || '' },
-        { tag: 'NextMarker', value: list.NextMarker },
         { tag: 'MaxKeys', value: listParams.maxKeys },
         { tag: 'Delimiter', value: listParams.delimiter },
         { tag: 'EncodingType', value: listParams.encoding },
         { tag: 'IsTruncated', value: isTruncated },
     ];
+    if (listParams.v2) {
+        xmlParams.push(
+            { tag: 'StartAfter', value: listParams.startAfter || '' });
+        xmlParams.push(
+            { tag: 'FetchOwner', value: `${listParams.fetchOwner}` });
+        xmlParams.push({
+            tag: 'ContinuationToken',
+            value: generateToken(listParams.continuationToken) || '',
+        });
+        xmlParams.push({
```

Comment:
Is the response always guaranteed to have a `NextContinuationToken`? In the sample response on the AWS docs, it looks like non-truncated responses do not have this key-value pair.

Reply:
Yup, that's right. It won't be included in the response unless it's present because of line 209